content/applications/sales/sales/send_quotations/quote_template.rst

```diff
@@ -49,89 +38,227 @@ Click the :guilabel:`Quotation Templates` link on the :guilabel:`Settings` page,
+terms, etc.) by clicking :guilabel:`Add a note`.
+
+.. image:: quote_template/lines-tab-quotation-template.png
+   :align: center
+   :alt: Populated lines tab on a quotation template form in Odoo Sales.
+
+To add a product to a quotation template, click :guilabel:`Add a product` in the :guilabel:`Lines`
+tab of a quotation template form. Doing so reveals a blank field in the :guilabel:`Product` column.
+
+When clicked, a drop-down menu with products from the database appear. Select the desired product
+from the drop-down menu to add it to the quotation template.
+
+.. tip::
+   If the desired product isn't readily visible, type the name of the desired product in the
+   :guilabel:`Product` field, and the option appears in the drop-down menu. Products can also be
+   found by clicking :guilabel:`Search More...` from the drop-down menu.
+
+.. note::
+   When a product is added to a quotation template, the default :guilabel:`Quantity` is `1`, but
+   that can be edited at any time.
+
+Then, drag-and-drop the product to the desired position, via the :guilabel:`six squares` icon,
+located to the left of each line item.
+
+To add a *section*, which serves as a header to organize the lines of a sales order, click
+:guilabel:`Add a section` in the :guilabel:`Lines` tab. When clicked, a blank field appears, in
+which the desired name of the section can be typed. When the name has been entered, click away to
+secure the section name.
+
+Then, drag-and-drop the section name to the desired position, via the :guilabel:`six squares` icon,
+located to the left of each line item.
+
+To add a note, which would appear as a piece of text for the customer on the quotation, click
+:guilabel:`Add a note` in the :guilabel:`Lines` tab. When clicked, a blank field appears, in which
+the desired note can be typed. When the note has been entered, click away to secure the note.
+
+Then, drag-and-drop the note to the desired position, via the :guilabel:`six squares` icon, located
+to the left of each line item.
+
+To delete any line item from the :guilabel:`Lines` tab (product, section, and/or note), click the
+:guilabel:`🗑️ (trash can)` icon on the far-right side of the line.
+
+Optional Products tab
+---------------------
+
+The use of optional products is a marketing strategy that involves the cross-selling of products
+along with a core product. The aim is to offer useful and related products to customers, which may
+result in an increased sale.
+
+For instance, if a customer wants to buy a car, they have the choice to order massaging seats, as
+well, or ignore the offer and simply buy the car. Presenting the choice to purchase optional
+products enhances the customer experience.
+
+Optional products appear as a section on the bottom of sales orders and eCommerce pages. Customers
+can immediately add them to their online sales orders themselves, if desired.
 
 In the :guilabel:`Optional Products` tab, :guilabel:`Add a line` for each compatible cross-selling
```

Comment:
"Compatible" might sound like some products are not eligible to be optional products if they're not compatible
```suggestion
In the :guilabel:`Optional Products` tab, :guilabel:`Add a line` for each cross-selling
```